src/solver.jl

```diff
@@ -87,6 +94,16 @@ function PajaritoSolver(;
     tol_prim_infeas = 1e-6,
     )
 
+    if mip_solver == UnsetSolver()
+        error("No MIP solver specified (set mip_solver)\n")
```

Comment:
I have a lot of errors in conic algorithm that tell users to try this or that with options. this is a lot easier to understand than those